tldw_Server_API/tests/Chat/integration/conftest.py

```
@@ -0,0 +1,82 @@
+    from tldw_Server_API.tests.Chat.integration.conftest_isolated import *  # noqa: F401,F403
+except Exception:
+    pass
+
+# Ensure OpenAI-backed tests use the local mock server to keep
+# Chat integration deterministic and avoid external dependencies.
+try:
+    import os
+    import pytest
+
+    @pytest.fixture(scope="session", autouse=True)
+    def _auto_configure_openai_mock(mock_openai_server):  # noqa: F811
```

Ruff (0.14.4)

Unused `noqa` directive (unused: `F811`)

Remove unused `noqa` directive

(RUF100)